process/app/main.py

```diff
@@ -0,0 +1,113 @@
+from flask import Flask
+
+import requests as rq
```

Comment:
You have some unused imports here I think? I can't see any of the following used:
```
logging
rq
datetime
csv
randint
```

Reply:
Good spot, thank you